specify/specify7
# Fix upload plan parser

Fixes #5305

The upload plan parser broke (again) with all the changes made in the data mapper to handle multiple trees. 

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions



- Go to workbench
- Create a new data set with taxon as the base table
- Select any tree
- Map any rank -> name
- Save
- Go back to Data Mapper
- [ ] Verify rank does not get unmapped
